apps/web/src/design-system/icons/general/Buildings.tsx

```diff
@@ -2,12 +2,19 @@ import React from 'react';
 /* eslint-disable */
 export function Buildings(props: React.ComponentPropsWithoutRef<'svg'>) {
   return (
-    <svg width="20" height="18" viewBox="0 0 20 18" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <svg
+      width="20"
+      height="18"
+      viewBox="0 0 20 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+      style={{ ...(props.style || {}), fill: props.style?.fill || 'currentColor' }}
```

Comment:
Would just adding color inside the Building component not work?

Reply:
it did not work for me but let me double-check it! :)